src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -6771,15 +6771,25 @@ End Class
             Dim intType As ITypeSymbol = comp.GetSpecialType(SpecialType.System_Int32)
 
             ' Illegal VB identifier and blank
-            Dim tuple2 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("123", ""))
-            Assert.Equal({"123", ""}, tuple2.TupleElementNames)
+            Dim tuple2 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("123", " "))
+            Assert.Equal({"123", " "}, GetTupleElementNames(tuple2))
 
             ' Reserved keywords
             Dim tuple3 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("return", "class"))
-            Assert.Equal({"return", "class"}, tuple3.TupleElementNames)
+            Assert.Equal({"return", "class"}, GetTupleElementNames(tuple3))
 
         End Sub
 
+        Private Shared Function GetTupleElementNames(tuple As INamedTypeSymbol) As ImmutableArray(Of String)
```

Comment:
This method preserves the existing logic of tests (that only look at user-provided names). 
Do we also have a test that verifies all the names of elements returned by the new API? For example, from `(int Alice, int, int Item3)`, we should get 3 elements, named "Alice", "Item2" and "Item3".